### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by rosskarchner**
## Changes

- more complete usage of the static tag in base.html (for the favicon and og:image)
- brings in a template bug fix that was included in 3.11.2
- remove's the CDN'ed jquery, per feedback from @ascott1 

## Review

- @cfpb/cfgov-backends 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by kave** [comment]
👍

**Comment by rosskarchner** [comment]
thanks, @kave ;)

File: cfgov/jinja2/v1/_layouts/base.html [hunk 1]
```diff
@@ -124,13 +123,7 @@
...
     <!--[if IE 9]><script src="{{ static('js/ie/common.ie.js') }}"></script><![endif]-->
 
-    {# TODO: jQuery CDN is provided in the head to satisfy
-             Google Tag Manager (GTM) requirements.
-             Ideally GTM would handle its own dependency management
-             and initializing jQuery could be moved to the footer. #}
-
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-    <script>if (!window.jQuery) { document.write('<script src="{{ static('js/jquery.min.js') }}"><\/script>'); }
+    <script src="{{ static('js/jquery.min.js') }}"></script>
```
Comment (Scotchester) [inline cfgov/jinja2/v1/_layouts/base.html]: Have we confirmed with the data team that they no longer require CDN-loaded jQuery in the `head`? And if they do, how do we reconcile that with the directive from @ascott1?

If they don't need it, then we should proceed, but move the jQuery inclusion to the bottom of `body`, right?

File: cfgov/jinja2/v1/newsroom/press-resources/index.html [hunk 2]
```diff
@@ -160,7 +160,7 @@ <h2 class="press-section_title press-photos-bios_title">Photos and bios</h2>
             <section class="contact-person contact-person__director content-l_col-1-2">
                 <div class="media">
                     <img class="media_image media_image__150 contact-person_photo"
-                         src="{{ static('img/director-cordray-round-300x300.jpg')}"
+                         src="{{ static('img/director-cordray-round-300x300.jpg')}}"
```
Comment (Scotchester) [inline cfgov/jinja2/v1/newsroom/press-resources/index.html]: Trivial: A space before the closing `{{` here would make this typo more obvious in the future :)

**Comment by sebworks** [comment]
I'm not sure why we aren't loading jQuery from the CDN. Most first time users will already have jQuery stored in their cache if we use the CDN.  I feel like we should still use the CDN and work on removing jQuery from the codebase and the Google Analytics code.

**Comment by Scotchester** [comment]
I wouldn't _think_ so, but, that comment in the HTML seems to indicate otherwise.

**Comment by rosskarchner** [comment]
@sebworks wrote:
> I'm not sure why we aren't loading jQuery from the CDN. Most first time users will already have jQuery stored in their cache if we use the CDN.  I feel like we should still use the CDN and work on removing jQuery from the codebase and the Google Analytics code. 

I think the fear is is that a CDN could be compromised, and then a hostile party could inject code into the website. Far fetched? Maybe, but @ascott1 (and security folks) seem to think it's a risk worth avoiding.

**Comment by sebworks** [comment]
> I'm not sure why we aren't loading jQuery from the CDN. Most first time users will already have jQuery stored in their cache if we use the CDN. I feel like we should still use the CDN and work on removing jQuery from the codebase and the Google Analytics code.

Yeah, but what about Google Analytics and Google Tag Manager? They say 70% of the web is using jQuery. I'd venture a significant portion of the group is loading jQuery from the Google CDN.  Their is also a performance hit for all our users once we have switched over. The minified file is 90kb.  I would rather work on removing jQuery from our codebase ( if possible given time constraints ).

**Comment by richaagarwal** [comment]
Curious about this.  I think I agree with @sebworks, but I'm wondering why the only concern with a CDN is the jQuery (if it were to be compromised)?  Not sure I'm following the logic here. If a CDN is compromised, don't we have other things to worry about as well?

**Comment by rosskarchner** [comment]
@richaagarwal in this case, we're talking about "ajax.googleapis.com" -- so the only problem would be jQuery. 

I 1000% agree with @sebworks (that the benefits of the CDN outweigh the risk), I just don't think it's a decision that's in our hands.

**Comment by rosskarchner** [comment]
I'm going to remove the jQuery change from this PR: from the discussion it's clearly not a "minor tweak" ;)

Internal issue 384 has been opened for further discussion.

**Comment by sebworks** [comment]
👍

File: cfgov/jinja2/v1/_layouts/base.html (hunk 1 above)
Comment (rosskarchner) [inline cfgov/jinja2/v1/_layouts/base.html]: I've removed this change!

File: cfgov/jinja2/v1/newsroom/press-resources/index.html (hunk 2 above)
Comment (rosskarchner) [inline cfgov/jinja2/v1/newsroom/press-resources/index.html]: done!

File: cfgov/jinja2/v1/_layouts/base.html [hunk 3]
```diff
@@ -74,8 +74,7 @@
         {% block og_article_author %}{% endblock %}
     <!-- End of Open Graph properties -->
 
-    {# TODO: Explicit favicon link needed for testing. Remove for production. #}
-    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="{{ static('assets/favicon.ico')}}">
```
Comment (rosskarchner) [inline cfgov/jinja2/v1/_layouts/base.html]: Done!

**Comment by rosskarchner** [comment]
it looks like Travis can't talk to Gitub right now--- maybe part of the big DNS outage?

**Comment by Scotchester** [comment]
Ah yes, that could be it.

**Comment by rosskarchner** [comment]
In a build [two hours ago](https://travis-ci.org/cfpb/cfgov-refresh/jobs/169589338):

> fatal: unable to access 'https://github.com/cfpb/cfgov-refresh.git/': Couldn't resolve host 'github.com'

**Pull request merged**